src/components/Modal/Modal.css

```diff
@@ -0,0 +1,95 @@
+
+.modal__wrapper {
+  font-family: Aktiv Grotesk, -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif;
+  position: absolute;
+  z-index: 1000;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+}
+
+.modal__wrapper.--hidden {
+  display: none;
+}
+
+.modal__overlay {
+  background: rgba(0, 0, 0, 0.6);
+  width: 100%;
+  height: 100%;
+  position: fixed;
+}
+
+.modal {
+  position: fixed;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  top: 50%;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  z-index: 2;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  max-width: 43.75rem;
+  max-height: 43.75rem;
+}
+
+@media screen and (max-width: 760px) {
+  .modal {
+    max-height: 100%;
+  }
+}
+
+.modal__content {
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+}
+
+.modal__header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0.6rem 1.2rem;
+}
+
+.modal__left--corner {
+  width: 20%;
+}
+
+.modal__close-button {
+  color: #2d2f34;
+  border: none;
+  background: transparent;
+  font-size: 1.45rem;
+  cursor: pointer;
+}
+
+.modal__center {
+  width: 60%;
+}
+
+.modal__title {
+  margin: 0 auto;
+  text-align: center;
+  color: #828282;
+  font-size: 1.1rem;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.modal__right--corner {
```

Comment:
In no parallel dimension can I see `right` as an element, and `corner` as a modifier. I think you meant `.modal__right-corner`? Or maybe `modal__header-right-corner`?